boards/arm/mimxrt1062_fmurt6/mimxrt1062_fmurt6_defconfig

```diff
@@ -0,0 +1,21 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright 2023 NXP
+#
+
+CONFIG_SOC_MIMXRT1062=y
+CONFIG_SOC_SERIES_IMX_RT=y
+CONFIG_BOARD_MIMXRT1062_FMURT6=y
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_GPIO=y
+CONFIG_ARM_MPU=y
+CONFIG_HW_STACK_PROTECTION=y
+CONFIG_PINCTRL=y
+CONFIG_FLASH=y
+
+# Enable Regulators
+CONFIG_REGULATOR=y
+CONFIG_REGULATOR_FIXED_INIT_PRIORITY=75
```

Comment:
Same comment as above for these regulator `Kconfigs`, why do we need this in the default?

Reply:
I need to enable all the regulators at boot time itself i.e. GPIOs controlling the Enable Line of board's Power sources - VDD_3V3_SD_CARD, VDD_5V_PERIPH, VDD_5V_HIPOWER, reg-3v3-sensor, VDD_3V3_SPEKTRUM, ETH_VDD_3V3.